motr/st/utils/sns_repair_common_inc.sh

```diff
@@ -34,13 +34,13 @@ prepare_datafiles_and_objects()
 			      -P $M0T1FS_PROC_ID -L ${lid} -s ${us} "
 
 		echo "creating object ${file[$i]} bs=${us} * c=${file_size[$i]}"
-		dd bs=${us} count=${file_size[$i]}            \
+		dd bs=${us} count="${file_size[$i]}"            \
 		   if="$MOTR_M0T1FS_TEST_DIR/srcfile"         \
 		   of="$MOTR_M0T1FS_TEST_DIR/src${file[$i]}"
 
-		$M0_SRC_DIR/motr/st/utils/m0cp ${MOTR_PARAM}     \
-						 -c ${file_size[$i]} \
-						 -o ${file[$i]}      \
+		"$M0_SRC_DIR/motr/st/utils/m0cp" "${MOTR_PARAM}"     \
```

Comment:
`${MOTR_PARAM}` should not be double quoted.

It is a list of parameters to be passed to m0cp/m0cat.

Reply:
Fixed.